src/state/selectors/utils.js

```diff
@@ -0,0 +1,4 @@
+/** */
```

Comment:
I don't know if this is a sufficient faux extension point to address #2931?  I'm thinking we can either leave it as-is as let consumers who want to move mirador do some webpacking magic, or we could make the default `state.mirador` and probably satisfy most use cases?